custom_components/frigate/camera.py

```diff
@@ -279,6 +284,79 @@ async def favorite_event(self, event_id: str, favorite: bool) -> None:
         await self._client.async_retain(event_id, favorite)
 
 
+class BirdseyeCamera(FrigateEntity, Camera):  # type: ignore[misc]
+    """Representation of the Frigate birdseye camera."""
+
+    # sets the entity name to same as device name ex: camera.front_doorbell
+    _attr_name = None
+
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        frigate_client: FrigateApiClient,
+    ) -> None:
+        """Initialize the birdseye camera."""
+        self._client = frigate_client
+        FrigateEntity.__init__(self, config_entry)
+        Camera.__init__(self)
+        self._url = config_entry.data[CONF_URL]
+        self._attr_is_on = True
+        # The device_class is used to filter out regular camera entities
+        # from motion camera entities on selectors
+        self._attr_device_class = DEVICE_CLASS_CAMERA
+        self._attr_is_streaming = True
+        self._attr_is_recording = False
+        self._stream_source = f"rtsp://{URL(self._url).host}:8554/birdseye"
```

Comment:
This needs to use the same `CONF_RTSP_URL_TEMPLATE` logic as the `CameraEntity` above, for people using reverse proxies.